src/Scientist/project.json

```diff
@@ -1,11 +1,12 @@
-﻿{
-  "version": "1.0.0-*",
+{
+  "version": "0.1.0-*",
```

Comment:
We can't do this at this point because 1.0.0-alpha is already up on NuGet. One thing we _could_ do is:

`1.0.0-alpha*`

After all, 1.0.0-alpha* is considered a lower version than 1.0.0. So I plan to just keep incrementing that until we're ready for the big 1.0.